docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/_index.md

```diff
@@ -64,7 +64,7 @@ Blog: [Using YugabyteDB xCluster DR for PostgreSQL Disaster Recovery in Azure](h
 
 ## Limitations
 
-- Currently, replication of DDL (SQL-level changes such as creating or dropping tables or indexes) is not supported. To make these changes requires first performing the DDL operation (for example, creating a table), and then adding the new object to replication in YugabyteDB Anywhere. In addition, xCluster does not support truncate operations. Refer to [Manage tables and indexes](./disaster-recovery-tables/).
+- Currently, replication of DDL (SQL-level changes such as creating or dropping tables or indexes) is not supported. For more details on how to propagate DDL changes from the DR primary to the DR replica, see [Schema change modes](#schema-change-mode-for-xcluster-dr).
```

Comment:
Having some issues with clicking on the #schema-change-mode-for-xcluster-dr url

Reply:
Also should we change `Currently, replication of DDL` to `Currently, automatic replication of DDL`